src/main/java/com/amihaiemil/versioneye/RtRepository.java

```diff
@@ -105,6 +105,7 @@ public boolean fork() {
                 branches.add(jsonBranches.getString(i));
             }
         } catch(final ClassCastException castException) {
+            throw new IllegalStateException();
```

Comment:
@amihaiemil Sorry but I don't understand what you mean by wrapping ClassCastException :) ?